### ISSUE TITLE
Is there anything wrong?

### ISSUE THREAD

**Issue by chenyanyin** [dvlshah/Re3#3]

**Comment by chenyanyin** [comment]
Hello, the tensorflow training code that you released in training folder is available? Could you tell me the result by Tensorflow weights and by Caffe weights is close?

**Comment by chenyanyin** [comment]
Could you release code that generate training file(labels.npy, image_names.txt,...)? I try to train it. If you reply to me, I will be grateful. Thank you~
